docs/streams/stream-basics/consumers.md

```diff
@@ -38,3 +38,60 @@ Client libraries can provide their own listener implementations for consumers. I
 ## Cursor
 
 A cursor is the subscription position for a stream consumer. You can think of it like the cursor on your page, marking where you are.
+
+## Create Consumers
+
+It is possible to use our JavaScript or Python SDK to create producers.
```

Comment:
```suggestion
It is possible to use our JavaScript or Python SDK to create consumers.
```
Maybe we can also rephrase this to something like.. "You can use our JavaScript or Python SDK to create consumers" But maybe James or Diana will have a better proposal for the rephrasing part.